test/postServiceTest.js

```diff
@@ -0,0 +1,18 @@
+/* global it, describe */
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+
+const PostService = require('../services/PostService');
+
+describe('PostService', () => {
+  describe('getOnePost(post_id)', () => {
+    it('should return correct post', () => {
+      const stub = sinon.stub(Post, 'find').returns(5);
```

Comment:
If you stub out a function you should assert that it is called with the right params

Reply:
added assertion